src/main/java/io/vertx/mqtt/MqttServerOptions.java

```diff
@@ -41,12 +41,17 @@
   public static final int DEFAULT_MAX_MESSAGE_SIZE = -1;
   public static final int DEFAULT_TIMEOUT_ON_CONNECT = 90;
 
+
+  public static final String MQTT_SUBPROTOCOL_CSV_LIST = "mqtt, mqttv3.1, mqttv3.1.1";
```

Comment:
As far as I can see from the IANA registry, the "official" one is just "mqtt" (which defines the latest version). Any specific reason you added the other ones?

Reply:
FYI: Paho Java client was using "mqttv3.1" before https://github.com/eclipse/paho.mqtt.java/pull/366, now it's only "mqtt"